lib/chewy/type/import.rb

```diff
@@ -150,8 +151,10 @@ def empty_objects_or_scope?(objects_or_scope)
 
         def import_linear(objects, routine)
           ActiveSupport::Notifications.instrument 'import_objects.chewy', type: self do |payload|
+            progressbar = ProgressBar.create total: adapter.import_count(objects) if routine.options[:progressbar]
```

Comment:
It would be nice to add a test case that mocks this progressbar object and make sure it has its progress incremented many times as expected